qipr_approver/approver/workflows/user_crud.py

```diff
@@ -63,10 +63,10 @@ def update_user_from_about_you_form(person, about_you_form, editing_user):
     person.last_name = about_you_form.get('last_name')
     person.webpage_url = about_you_form.get('webpage_url')
     person.title = about_you_form.get('title')
-    person.department = about_you_form.get('department')
     person.qi_required = about_you_form.get('qi_required')
     person.training = about_you_form.get('training_program')
     person.self_classification = extract_model(Self_Classification, "name", about_you_form.get('select-self_classification') or '')
+    person.department = extract_model(ClinicalDepartment, "name", about_you_form.get('select-department') or '')
```

Comment:
Can you put this after the following if statement since the if statement is related to the above self-classification? Also, if we decide to add a "other department" then we need to add the same if block as below (or make a function)